src/router/index.ts

```diff
@@ -34,5 +34,11 @@ export default route(function (/* { store, ssrContext } */) {
     history: createHistory(process.env.VUE_ROUTER_BASE),
   });
 
+  Router.beforeEach((to, from) => {
```

Comment:
Please remove unused variable

```suggestion
  Router.beforeEach((to) => {
```